scripts/modules/elastic_stack.py

```diff
@@ -42,14 +42,8 @@ def __init__(self, **options):
                 kibana_url = kibana_scheme + "://admin:changeme@" + self.DEFAULT_KIBANA_HOST
             self.depends_on = {"kibana": {"condition": "service_healthy"}}
 
-            xpack_registry_url = "https://epr-snapshot.elastic.co"
-            if options.get("enable_package_registry"):
-                xpack_registry_url = "http://package-registry:{}".format(PackageRegistry.SERVICE_PORT)
-                self.depends_on["package-registry"] = {"condition": "service_healthy"}
```

Comment:
Did you remove the healthcheck on purpose?

Reply:
Yes:

> Remove the dependency from APM Server to the local package registry; there is already a transitive dependency through Kibana.